runtime/frequency/src/development_genesis.rs

```diff
@@ -0,0 +1,154 @@
+use common_primitives::node::Verify;
+use cumulus_primitives_core::ParaId;
+use sp_core::{sr25519, Pair, Public};
+use sp_runtime::traits::IdentifyAccount;
+#[cfg(not(feature = "std"))]
+use sp_std::alloc::format;
+
+use crate::*;
+
+// Generate the session keys from individual elements.
+//
+// The input must be a tuple of individual keys (a single arg for now since we have just one key).
+fn template_session_keys(keys: AuraId) -> SessionKeys {
+	SessionKeys { aura: keys }
+}
+
+#[allow(clippy::unwrap_used)]
+fn load_genesis_schemas() -> Vec<crate::pallet_schemas::GenesisSchema> {
+	serde_json::from_slice(include_bytes!("../../../resources/genesis-schemas.json")).unwrap()
```

Comment:
Does anyone have any opinion on moving this file out of resources into a new location?

Reply:
seems fine where it is to me